modules/social_features/social_search/social_search.install

```diff
@@ -28,59 +28,40 @@ function social_search_update_dependencies() {
  * Perform actions related to the installation of social_search.
  */
 function social_search_install() {
-
-  // Set some default permissions.
-  _social_search_set_permissions();
-}
-
-/**
- * Function to set permissions.
- */
-function _social_search_set_permissions() {
-  $roles = Role::loadMultiple();
-
-  /** @var \Drupal\user\Entity\Role $role */
-  foreach ($roles as $role) {
-    if ($role->id() === 'administrator') {
-      continue;
-    }
-
-    $permissions = _social_search_get_permissions($role->id());
-    user_role_grant_permissions($role->id(), $permissions);
-  }
-}
-
-/**
- * Return the permissions per role.
- *
- * @param string $role
- *   The role to get the permissions for.
- *
- * @return array
- *   A list of permissions.
- */
-function _social_search_get_permissions($role) {
-  // Anonymous.
-  $permissions['anonymous'] = [
-    'access content search',
-  ];
-
-  // Authenticated.
-  $permissions['authenticated'] = array_merge($permissions['anonymous'], [
-    'access users search',
-    'access group search',
-  ]);
-
-  // Content manager.
-  $permissions['contentmanager'] = array_merge($permissions['authenticated'], []);
-
-  // Site manager.
-  $permissions['sitemanager'] = array_merge($permissions['contentmanager'], []);
-
-  if (isset($permissions[$role])) {
-    return $permissions[$role];
-  }
-  return [];
+  // Grant the default permissions for this feature.
+  user_role_grant_permissions(
+    RoleInterface::ANONYMOUS_ID,
+    [
+      'access content search',
+    ]
+  );
+  user_role_grant_permissions(
```

Comment:
At `social_group.install` we are giving roles (including anonymous and authenticated) `access group search`.

https://github.com/goalgorilla/open_social/pull/2377/files#diff-5f7ae7c82ce8648848b85169972250edb5ef83a6c4add7be625d1ccebcb61dbcR57

Reply:
Let's not duplicate this, maybe best if we remove it at `social_group`?

Reply:
I have removed giving `access group search` in `social_group` and have moved it to `social_search`.